classes/book_class.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'rental_class'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Dear @kerlos-bahaa, Good job setting up the methods and working through the association. But when we add a rental, instead of accepting a `rental` as a parameter, we need to accept the `person`, and `date` as a parameter and create the rental as a new one with the book being `self`, creating a new rental will add the rental to the `book.rentals` inside its initializer method. This way of doing this helps us preserve the association between `rental` and the `book`.

> something like `Rental.new(date, self, person)`

> Remember if we don't pass `self` as a parameter, there is no way of guaranteeing we are working with this `book`.

Reply:
This change has been done well